docs/introduction-to-smart-contracts.rst

```diff
@@ -12,9 +12,6 @@ Let us begin with a basic example that sets the value of a variable and exposes
 it for other contracts to access. It is fine if you do not understand
 everything right now, we will go into more detail later.
 
-Storage
```

Comment:
This is meant to say: First simple example on storage. The next will be a subcurrency.